### ISSUE TITLE
review landuse mapping tables

### ISSUE THREAD

**Issue by DirkEilander** [Deltares/hydromt_wflow#84]
try to make the landuse mapping tables consistent 
add esa worldcover mapping table

**Comment by markhegnauer** [comment]
Is someone working on this? I want to use ESA WorldCover and made an initial mapping table (based on VITO mapping table). WorldCover comes with less classes, mostly they do not distinguish between forest classes. Below my proposed mapping compared to VITO:

![image](https://user-images.githubusercontent.com/33261105/194005110-fe8b8f4a-e48b-4d1b-9d86-140139312316.png)

**Comment by hboisgon** [comment]
ESA worlcover was added but still good to review values between the different landuse classification

**Comment by hboisgon** [comment]
One source: USDA (1986) Urban hydrology for small watersheds, second ed, technical release 55. United states Department of Agriculture. Natural Resources Conservation Service. Conservation Engineering Division, Washington, D.C

![Image](https://github.com/user-attachments/assets/a83ba881-0bdd-4631-a80d-25094b2ef416)

**Comment by shartgring** [comment]
We are using OSM land use land cover for JCAR ATRACE (https://osmlanduse.org/, https://heidata.uni-heidelberg.de/dataset.xhtml?persistentId=doi:10.11588/data/IUTCDN). I also did some quick mapping selection based on CORINE land cover but I would be happy to improve it and include it if better sources are available

**Comment by atsiokanos** [comment]
Is this still in your to-do list? I think it would be nice to review the landuse related params for v1 as recently I discovered some suspicious values (e.g rooting depth for forest etc.)

**Comment by hboisgon** [comment]
Yes it's in the release candidate list :) I put it in the list because there are more values than this that are suspicious. If you have any literature or references for this I would be happy to have it! I cannot really find where the original values (for globcover) came from

**Comment by JoostBuitink** [comment]
One thing that stood out while reviewing a custom land use mapping table: `vegetation_kext` is very low for the shrub-classes in the below default mapping tables. I think this is supposed to be 0.7 instead of the current 0.07.
- ESA (class 20)
- GLCNMO (class 7)
- Vito (class 20)

**Comment by hboisgon** [comment]
Just discussed that it would be good to still keep the old default tables and use data catalog versioning to create and add the new ones.
We will for v1 focus mainly on harmonising values between land use data sources and reviewing some of the strange values like ``k_ext`` or ``crop_factor``.

Finding "better" values eg for manning roughness can then be done in a version 2 if time does not allow.
